client_test.go

```diff
@@ -479,8 +477,12 @@ func TestGetFeature(t *testing.T) {
 	// Given
 	masterAPIKey := "master_api_key"
 	featureUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+	projectUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
```

Comment:
We should probably use different UUIDs for feature / project to avoid confusion

Reply:
Yeah, good point